catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -45,9 +51,15 @@
 
     private CswAxisOrder cswAxisOrder;
 
+    private WKTReader wktReader;
+
+    private WKTWriter wktWriter;
+
     public BoundingBoxReader(HierarchicalStreamReader reader, CswAxisOrder cswAxisOrder) {
         this.reader = reader;
         this.cswAxisOrder = cswAxisOrder;
+        wktWriter = new WKTWriter();
```

Comment:
Don't we need to be careful about the (lack of) thread safety of the writer?  https://github.com/codice/ddf/blob/08651d5cc59e559b940b4faaec6334657948c18c/catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java#L101